store/store.go

```diff
@@ -289,11 +289,15 @@ func assertsURL() string {
 	if u := os.Getenv("SNAPPY_FORCE_SAS_URL"); u != "" {
 		return u
 	}
-	if useStaging() {
-		return "https://assertions.staging.ubuntu.com/v1/"
+	storeBaseURI, err := url.Parse(apiURL())
+	if err != nil {
+		panic(err)
 	}
-
-	return "https://assertions.ubuntu.com/v1/"
+	assertsURI, err := storeBaseURI.Parse("api/v1/snaps/")
```

Comment:
URL.Parse is actually not ideal for this use case, i.e. where we only ever want to extend the path. I think using a simple helper here and in similar places would be more robust, e.g.:

    func urlJoin(base *url.URL, paths ...string) *url.URL {
        if len(paths) == 0 {
            return base
        }
        url := *base
        url.RawQuery = ""
        for _, morePath := range paths {
            url.Path = path.Join(url.Path, morePath)
        }
        return &url
    }

It also has the advantage of being impossible to error, I think.